skin/src/build.gradle

```diff
@@ -37,6 +37,7 @@ android {
     }
 
     resourcePrefix 'rss_'
+    sourceSets { main { java.srcDirs = ['src/main/java', 'src/test/java'] } }
```

Comment:
Okay, looking at this line more closely, it seems that you added a test for the Task step, but didn't actually include that test in the commit? @nrlakin Can give you some guidance on where to include the test so that it'll be picked up and run by Travis, if you need any advice on that.